finality-aleph/Cargo.toml

```diff
@@ -21,7 +21,7 @@ sp-keystore = { git = "https://github.com/paritytech/substrate.git", rev = "1fa8
 sc-network = { git = "https://github.com/paritytech/substrate.git", rev = "1fa8cf7"}
 sc-network-gossip = { git = "https://github.com/paritytech/substrate.git", rev = "1fa8cf7"}
 sc-telemetry = { git = "https://github.com/paritytech/substrate.git", rev = "1fa8cf7"}
-rush = { git = "https://github.com/Cardinal-Cryptography/rush.git", rev = "caed0da" }
+rush = { git = "https://github.com/Cardinal-Cryptography/AlephBFT.git", rev = "381d388" }
```

Comment:
Is the plan to rename rush -> alephbft in a separate PR? Otherwise you can do it here already.

Reply:
Doesn't work, at this commit there is no aleph-bft crate in the AlephBFT repository.